notifications/src/main/java/org/openedx/notifications/presentation/primer/NotificationsPrimerViewModel.kt

```diff
@@ -43,4 +43,8 @@ class NotificationsPrimerViewModel(
     fun dismissDialog() {
         _uiState.value = PrimerUIState.DismissDialog
     }
+
+    fun showRationalDialog() {
```

Comment:
A small typo. It's `Rationale`. Please fix in other places as well.

Reply:
addressed.